ffi/harfbuzz_coverage.lua

```diff
@@ -0,0 +1,347 @@
+-- generated by https://github.com/ezdiy/unimap/blob/master/gen.lua
```

Comment:
Can you add a few comments here describing what these 2 datastructures and their contents are? And what the input to your gen.lua is?
Just so it's a bit less black magic, we know where that comes from, and if/when it needs to be updated.

(Was that unimap some previous project of yours ? Or did you cook it up just for this ?)

Reply:
Unimap is ported from different lua thing, yes. I'm not all that thrilled how the table is thrown in here, but the alternative seems even more spammy - makefile/wget Scripts.txt/gen.lua somewhere in koreader make, for a table that's quite unlikely to ever change. Alas, wouldn't describing the format make more sense on harfbuzz.lua end?

Reply:
> Alas, wouldn't describing the format make more sense on harfbuzz.lua end?

Whereever you feel it is best - with just a backlink (`-- see harfbuzz.lua for details`) in this file so we know where to look.
Just a few words about what this is all about - so one can get what it's about and just skip it as _ok, not interesting_ :)